libs/features/personalization/personalization.js

```diff
@@ -127,6 +127,10 @@ const createFrag = (el, url, manifestId) => {
   const a = createTag('a', { href }, url);
   if (manifestId) a.dataset.manifestId = manifestId;
   let frag = createTag('p', undefined, a);
+  const isDelayedModalAnchor = /#.*delay=/.test(href);
```

Comment:
could you add a test case for it in personalization.js please? that it adds 'hide-block' class